examples/app-with-extension-point/package.json

```
@@ -5,8 +5,7 @@
   "scripts": {
     "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
     "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
-    "e2e:update": "npm exec cypress install && npm exec grafana-e2e run --update-screenshots",
-    "e2e": "npm exec cypress install && npm exec grafana-e2e run",
+    "e2e": "npx playwright test",
```

```suggestion
    "e2e": "playwright test",
```

I don't think we need to run it via npx